pkg/logs/config/integration_config.go

```diff
@@ -94,33 +98,36 @@ func buildLogsSources(ddconfdPath string) ([]*IntegrationConfigLogSource, error)
 			log.Error(err)
 			continue
 		}
-
+		integrationName := strings.TrimSuffix(file, filepath.Ext(file))
```

Comment:
is there some shared logic in the agent for that? The day it changes we won't think of this

Reply:
There is none for now, I think it's up to us to decide how to compute the name of our integration, WDYT ?

Reply:
What I mean is that there is already some logic in the agent to do that. I would do a small refactoring to create some shared logic for that and use it, as we will follow the agent for this, and if one they we change how we configure integrations in the agent we won't think of updating this